kubernetes/hmda-regulator/values.yaml

```diff
@@ -0,0 +1,63 @@
+# Default values for check-digit.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: hmda/hmda-regulator
+  tag: v2.7.2
+  pullPolicy: IfNotPresent
+
+service:
+  type: NodePort
+  port: 9099
+  targetPort: 9099
+  name: http-regulator
+
+grpc:
+  type: ClusterIP
+  port: 60082
+  targetPort: 60082
+  name: http2-regulator
+
+ambassador:
+  name: ambassador-regulator
+  port: 80
+  service:
+    type: NodePort
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  ContainerPort: 9099
+  hosts:
+    - hmda-regulator.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
```

Comment:
Let's put in some resource constraints here. Maybe 4 gbs of memory for each limits and requests and then 1 cpu request with 2 cpu limits.